sell-loot.lic

```diff
@@ -76,11 +76,18 @@ class SellLoot
     fput("open my #{container}")
     return if inside?("#{adj} pouch", container)
     walk_to @hometown['gemshop']['id']
-    @hometown['gemshop']['name'] = "Veilex" if DRRoom.npcs.include?("Veilex") # Shard gemshop changes person manning the shop
-    fput("ask #{@hometown['gemshop']['name']} for #{adj} pouch")
+    clerk = which_clerk(@hometown['gemshop']['name'])
+    fput("ask #{clerk} for #{adj} pouch")
     fput("put my pouch in my #{container}")
   end
 
+  def which_clerk(clerks)
+    return clerks if clerks.is_a? String
+
+    clerks.select! { |clerk| DRRoom.npcs.include?(clerk) }
```

Comment:
So, just to put my thoughts out there... Since this is semistructured data, we dont have to make every attribute a list because of one outlier. We have to adapt the program to handle this outlier in either case. However the data entry becomes much easier if its just a field for a string, which i think all the other gemshops are.